### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by a17t**
Just some initial changes to the readme file. What do you think?

### CODE REVIEW

**Comment by yschroeder** [comment]
Looks good as a first shot 👍

**Comment by gador** [comment]
LGTM. Later on we should switch to our own readthedocs.

File: ansible/meta/main.yml [hunk 1]
```diff
@@ -13,5 +13,5 @@ galaxy_info:
     - name: Ubuntu
       versions:
         - focal
-
+  role_name: bare_metal_paperless
```
Comment (bauerj) [inline ansible/meta/main.yml:16]: How is this related?

Comment (a17t) [reply ansible/meta/main.yml:16]: its my try on fixing the ansible related checks for the PR. (Although currently that did not help.)

**Comment by a17t** [comment]
Ok I reverted all the ansible check related changes. I initialy thought it might be a quick fix from the original error message, but its much more complex.

File: ansible/meta/main.yml (hunk 1 above)
Comment (a17t) [reply ansible/meta/main.yml:16]: reverted those changes as those ansible tests are much more complex than expected.

**Comment by bauerj** [comment]
Can you remove the ansible-related changes from this PR then?

And please open an issue stating that this is broken right now.

**Comment by a17t** [comment]
> And please open an issue stating that this is broken right now.

I was just in the process of doing so.

**Review by bauerj — APPROVED** [review]

**Comment by bauerj** [comment]
Thanks!
